apps/voting/contracts/Voting.sol

```diff
@@ -24,6 +24,7 @@ contract Voting is IForwarder, AragonApp {
 
     uint64 public constant PCT_BASE = 10 ** 18; // 0% = 0; 1% = 10^16; 100% = 10^18
 
+    uint256 private constant UINT_96_MAX = 0xFFFFFFFFFFFFFFFFFFFFFFFF;
```

Comment:
Why not `uint96(-1)` to not manually count the number of `F`s

Reply:
You're right; this way of declaring the number could seem confusing. I'll fix it.

Reply:
Fixed in 5ef1358d79f3f47f136067ccf41c4bef9484179e